event/bus.go

```diff
@@ -30,11 +35,14 @@ type Bus interface {
 	// last send to the channel.
 	//
 	// Example:
-	// ch := make(chan EventT, 10)
-	// defer close(ch)
-	// cancel, err := eventbus.Subscribe(ch)
-	// defer cancel()
-	Subscribe(typedChan interface{}, opts ...SubscriptionOpt) (CancelFunc, error)
+	// sub, err := eventbus.Subscribe(new(EventType))
+	// defer sub.Close()
+	// for e := range sub.Out() {
+	//   event := e.(EventType) // guaranteed safe
+	//   [...]
+	// }
+	// TODO: update doc
+	Subscribe(eventType interface{}, opts ...SubscriptionOpt) (Subscription, error)
```

Comment:
Shouldn't this be `[]interface{}`.

Ahh, it accepts both.
This should be documented.